scripts/webpack.config.site.js

```diff
@@ -1,37 +0,0 @@
-import { fileURLToPath } from 'url'
```

Comment:
[mountain] It'd be good to delete the old site in a separate PR since the old site currently isn't being used and this PR can be just the new work.

Reply:
Done in https://github.com/netlify/framework-info/pull/803